test/base_mods/MM-chebi.sam

```diff
@@ -0,0 +1,2 @@
+@CO	Separate m, h and N modifications
+*	0	*	0	0	*	*	0	0	AGCTCTCCAGAGTCGNACGCCATYCGCGCGCCACCA	*	Mm:Z:C+m,2,2,1,4,1;C+76792,6,7;N+n,15;	Ml:B:C,102,128,153,179,204,161,187,212,169
```

Comment:
Micro-quibble.  There are more Ml entries than Mm ones.  The final 169 value is ignored.